web/src/features/map/mapAtoms.ts

```diff
@@ -11,3 +11,5 @@ export const mousePositionAtom = atom({
 export const hoveredZoneAtom = atom<HoveredZone | null>(null);
 
 export const mapMovingAtom = atom(false);
+
+export const zoneIdAtom = atom<FeatureId>(undefined);
```

Comment:
Hmm, could we name this so that it is more clear what this is? Perhaps something like "activeZoneId" or "selectedZoneId"?

Reply:
Have updated this